code/index.html

```diff
@@ -4,19 +4,139 @@
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lobster&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <header class="header container">
+      <a href="#"><div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div></a>
+      <a href="#"><h1 class="top-heading">Fizzy TechNews</h1></a>
+      <img class="logo" src="./logo.svg" alt="Technigo logo">
+      <nav class="navigation-container">
+        <a class="navigation-item" href="#">Top News</a>
+        <a class="navigation-item" href="#">Latest News</a>
+        <a class="navigation-item" href="#">Industry</a>
+        <a class="navigation-item" href="#">Contact Us</a>
+      </nav>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
+    <section class="big-news container">
+      <div class="news-card">
+        <a class="news-link" href="#">
+          <div class="polaroid"><img class="news-card-pic" src="https://images.unsplash.com/photo-1524289286702-f07229da36f5?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1950&q=80">
+            <div class="big-news-text">
+              <p class="title">Lorem Ipsum</p>
+              <p class="paragraph">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p> 
+            </div>
+          </div>
+        </a>
+      </div>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <a href="#"><section class="news-type container">
+      <a href="#"><p class="type">Top News</p></a>
+    </section></a>
```

Comment:
An `a` tag in an `a` tag doesn't make much sense (the inner one will not be clickable). Maybe you can scale this down to:
`<a href="#" class="type">Top News</a>`
and then style it as you want in the CSS

Reply:
Goes for all occurences below as well